### PR TITLE
Refactor components

### PR DESCRIPTION
**Pull request by GideonJagen**
Refactor components, file structure and use of static methods for creating widgets

### CODE REVIEW

File: components/Reset_and_search.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        dbc.Col(children=[Reset_and_search.search_button()]),
+        ])],
+        style = {
+        'width' : '500px'
+        }
+        )
+        return widget
+
+    @staticmethod
+    def reset_filter_button():
```
Comment (raoulkent) [inline components/Reset_and_search.py]: should be `_reset_filter_button()`

File: components/Reset_and_search.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        'height' : '75px',
+        'font-size': '16px',
+        'background-color' : '#CC3311'
+        }
+        )
+        return widget
+
+
+    @staticmethod
+    def search_button():
```
Comment (raoulkent) [inline components/Reset_and_search.py]: should be `_search_button()`

File: components/opTime_slider.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            id='opTime_widget',
+            children=[
+                html.H4('Operationstid'),
+                OpTime_widget.opTime_slider(min_time, max_time),
+            ]
+        )
+        return widget
+
+    @staticmethod
+    def opTime_slider(min_time, max_time):
```
Comment (raoulkent) [inline components/opTime_slider.py]: should be `_opTime_slider(min_time, max_time)`

File: components/statistics_code.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            id='statistics_code_widget',
+            children=[
+                html.H4('Statistikkod'),
+                Statistics_code_widget.statistics_code_dropdown()
+            ]
+        )
+        return widget
+
+    @staticmethod
+    def statistics_code_dropdown():
```
Comment (raoulkent) [inline components/statistics_code.py]: should be `_statistics_code_dropdown()`

**Review by davidjohnsson96 — APPROVED** [review]

**Review by raoulkent — COMMENTED** [review]
Just a few methods that should be `_private()`

**Review by raoulkent — APPROVED** [review]
Looks good!

**Pull request merged**